src/sentry/services/hybrid_cloud/organizationmember_mapping/__init__.py

```diff
@@ -43,14 +43,14 @@ class RpcOrganizationMemberMappingUpdate(TypedDict, total=False):
     invite_status: Optional[int]
 
 
-def update_organizationmember_mapping_from_instance(
+def rpc_update_organizationmember_mapping_from_instance(
     organization_member: OrganizationMember,
 ) -> RpcOrganizationMemberMappingUpdate:
     attributes = {
         attr_name: getattr(organization_member, attr_name)
         for attr_name in RpcOrganizationMemberMappingUpdate.__annotations__.keys()
     }
-    return RpcOrganizationMemberMappingUpdate(**attributes)  # type: ignore
+    return RpcOrganizationMemberMappingUpdate(**attributes)
```

Comment:
Should we have `RpcOrganizationMemberMappingUpdate.from_orm()` We have methods like that for user and they are working well.